odkx-src/survey-using.rst

```diff
@@ -264,6 +264,22 @@ Basic instructions for designing Survey forms are provided in the :ref:`build-ap
 
 Survey forms are created in :program:`Excel` and saved as :file:`.xlsx` files. These are converted into form definitions using the :doc:`xlsx-converter-intro`. The linked guide should help you create and modify the files to create your own forms.
 
+**Some text formatting options in Survey:**
```

Comment:
I would move this added content out of "Using survey" and into the XLSX converter reference, since this is something you do in XLSX converter not in survey itself.